opennlp-docs/src/docbkx/tokenizer.xml

```diff
@@ -258,37 +258,39 @@ Arguments description:
 				To train the english tokenizer use the following command:
 				<screen>
 			    <![CDATA[
-$ opennlp TokenizerTrainer -model en-token.bin -alphaNumOpt -lang en -data en-token.train -encoding UTF-8
+$ opennlp TokenizerTrainer -model en-token.bin -alphaNumOpt isAlphaNumOpt -lang en -data en-token.train -encoding UTF-8
```

Comment:
Looking at this again, should this be `true` instead of `isAlphaNumOpt`? Like:

```
$ opennlp TokenizerTrainer -model en-token.bin -alphaNumOpt true -lang en -data en-token.train -encoding UTF-8
```

Looking at `TrainingParams.java`, `getAlphaNumOpt()` returns a `Boolean` and defaults to `false`. Since it defaults to `false`, I guess it makes sense for the example to be `true`...?

Reply:
Thank you for taking a notice to this @jzonthemtn , I certainly see the difference :

`$opennlp TokenizerTrainer -model en-token-test.bin -alphaNumOpt true -lang en -data en-token.train -encoding UTF-8 -cutoff 5
Indexing events with TwoPass using cutoff of 5

	Computing event counts...  done. 45 events
	Indexing...  done.
Sorting and merging events... done. Reduced 45 events to 25.
Done indexing in 0,09 s.
Incorporating indexed data for training...
done.
	Number of Event Tokens: 25
	    Number of Outcomes: 2
	  Number of Predicates: 18
...done.



$opennlp TokenizerTrainer -model en-token-test.bin -alphaNumOpt false -lang en -data en-token.train -encoding UTF-8 -cutoff 5
Indexing events with TwoPass using cutoff of 5

	Computing event counts...  done. 212 events
	Indexing...  done.
Sorting and merging events... done. Reduced 212 events to 171.
Done indexing in 0,12 s.
Incorporating indexed data for training...
done.
	Number of Event Tokens: 171
	    Number of Outcomes: 2
	  Number of Predicates: 75
...done.`

Worth updating the doc.